### ISSUE TITLE
Support disable Dota2 BGM

### ISSUE THREAD

**Issue by robincodex** [ModDota/BugTracker#117]
As the title says.

I used my own music, but it was mixed with dota2 music.

**Comment by ChalkyBrush** [comment]
best solution for now is to add an option for players to mute your game's music

**Comment by robincodex** [comment]
@ChalkyBrush yeah, best can also support.

**Comment by Arhowk** [comment]
You can import a blank wav file to override the default soundtrack

**Comment by ChalkyBrush** [comment]
you'd have to do it for EVERY soundtrack :D
and keep it up to date by adding blank wavs every time a new soundtrack is added

But that is the most complete solution

**Comment by robincodex** [comment]
@Arhowk yeah, I has override all music to blank wav, but it may not be override.

**Comment by DoctorGester** [comment]
Everything is supported already. Starting the soundevent of a proper type mutes the default music alright. Study the default music soundevents files.
